<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>740</y>
      <width>1131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QGroupBox {
    background-color: white;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1131</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="stop_button">
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: white; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/stop.png</normaloff>:/img/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="play_button">
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: white; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/play.jpg</normaloff>:/img/play.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+P</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="up_button">
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: white; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/uparrw-Copy.png</normaloff>:/img/uparrw-Copy.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+U</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="down_button">
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: white; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/dwnarrw.png</normaloff>:/img/dwnarrw.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="left_button">
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: white; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/larrw.png</normaloff>:/img/larrw.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+L</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="right_button">
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: white; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/Rarrw.png</normaloff>:/img/Rarrw.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoom_out">
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: white; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/zoomout.JPG</normaloff>:/img/zoomout.JPG</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+O</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoom_in">
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: white; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/zoomin.JPG</normaloff>:/img/zoomin.JPG</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+I</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>798</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="PlotWidget" name="graphicsView_1">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>0</y>
      <width>1141</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>1131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>1131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>250</y>
      <width>1141</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>1141</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>1131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1134</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionchannel_1"/>
     <addaction name="actionChannel_2"/>
     <addaction name="actionChannel_3"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionPrint_to_PDF"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPrint_to_PDF">
   <property name="text">
    <string>Print to PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionchannel_1">
   <property name="text">
    <string>Channel 1</string>
   </property>
  </action>
  <action name="actionChannel_2">
   <property name="text">
    <string>Channel 2</string>
   </property>
  </action>
  <action name="actionChannel_3">
   <property name="text">
    <string>Channel 3</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
